tests/jvm/src/test/scala/scala/meta/tests/cli/CliSuite.scala

```diff
@@ -12,7 +12,9 @@ import scala.meta.testkit.DiffAssertions
 class CliSuite extends BaseCliSuite {
   val sourceroot = Files.createTempDirectory("sourceroot_")
   val helloWorldScala = sourceroot.resolve("HelloWorld.scala")
-  Files.write(helloWorldScala, """
+  Files.write(
+    helloWorldScala,
+    """
```

Comment:
Am I right in thinking that your setup automatically applies scalafmt to all files that you're editing, not just to those that are mentioned in `.scalafmt.conf`?